versioned_docs/version-8.3/reference/dependencies.md

```diff
@@ -2897,16 +2897,16 @@ Desktop Modeler is a desktop modeling application that builds upon a number of t
 - [io.undertow:undertow-core@2.3.12.Final](http://www.jboss.org/undertow-parent/undertow-core) ([Apache License Version 2.0](http://repository.jboss.org/licenses/apache-2.0.txt))
 - [io.undertow:undertow-servlet@2.3.12.Final](http://www.jboss.org/undertow-parent/undertow-servlet) ([Apache License Version 2.0](http://repository.jboss.org/licenses/apache-2.0.txt))
 - [io.undertow:undertow-websockets-jsr@2.3.12.Final](http://www.jboss.org/undertow-parent/undertow-websockets-jsr) ([Apache License Version 2.0](http://repository.jboss.org/licenses/apache-2.0.txt))
-- [jakarta.activation:jakarta.activation-api@2.1.2](https://github.com/jakartaee/jaf-api) ([EDL 1.0](http://www.eclipse.org/org/documents/edl-v10.php))
-- [jakarta.annotation:jakarta.annotation-api@2.1.1](https://projects.eclipse.org/projects/ee4j.ca) ([EPL 2.0](http://www.eclipse.org/legal/epl-2.0))
+- [jakarta.activation:jakarta.activation-api@2.1.3](https://github.com/jakartaee/jaf-api) ([EDL 1.0](http://www.eclipse.org/org/documents/edl-v10.php))
+- [jakarta.annotation:jakarta.annotation-api@2.1.1](https://projects.eclipse.org/projects/ee4j.ca) ([EPL 2.0](http://www.eclipse.org/legal/epl-2.0)) and ([GPL2 w/ CPE](https://www.gnu.org/software/classpath/license.html))
```

Comment:
❌ We should keep just one license, EPL in this case

Reply:
good catch, I missed this one